components/servo/build.rs

```diff
@@ -17,22 +17,46 @@ fn main() {
 
 fn android_main() {
     // Get the NDK path from NDK_HOME env.
-    let ndk_path = env::var("NDK_HOME").ok().expect("Please set the NDK_HOME environment variable");
+    let ndk_path = env::var("ANDROID_NDK").ok().expect("Please set the ANDROID_NDK environment variable");
     let ndk_path = Path::new(&ndk_path);
 
-    // Get the standalone NDK path from NDK_STANDALONE env.
-    let standalone_path = env::var("NDK_STANDALONE").ok().expect("Please set the NDK_STANDALONE environment variable");
-    let standalone_path = Path::new(&standalone_path);
+    // Build up the path to the NDK compilers
+    // Options for host are:  "linux-x86_64" "linux-x86" "darwin-x86_64" "darwin-x86"
```

Comment:
Is `darwin-x86` a relevant option? I don't see it listed below, although it is mentioned in other places.